### DISCUSSION TITLE
Can we customize existing menu to support file/image attachment from local system

### DISCUSSION THREAD

**Discussion by piyush0966** [sibiraj-s/ngx-editor#452] [Q&A] [answered]
How can we attach image from our system into the rich text editor? Can we create a custom menu that supports it or is it possible to update the existing image pop up to support file attachments?

**Comment by sibiraj-s** [comment, marked as answer]
Hi. @piyush0966 . You cannot customize the existing image popup, but you can create a custom menu component. See https://sibiraj-s.github.io/ngx-editor/en/menu/#custom-menu

**Reply by pramodpadmanabhi** [reply]
@sibiraj-s  The link seems to be broken share it again please!

**Reply by sibiraj-s** [reply]
https://sibiraj-s.github.io/ngx-editor/menu/#custom-menu. Here is the one, the i18n setup redirect seems broken, will fix them
